recOrder/io/_reader.py

```diff
@@ -16,7 +16,7 @@ def napari_get_reader(path):
 def ome_zarr_reader(
     path: Union[str, List[str]]
 ) -> List[Tuple[zarr.Array, Dict]]:
-    reader = WaveorderReader(path)
+    reader = read_micromanager(path)
```

Comment:
This does not open the files recOrder acquires since they are not HCS.

Reply:
Thanks, good call. Following our discussion, I appreciate your help in extending the recOrder reader to handle hcs formats.

Reply:
See c8be8910c4a8b61e3e4ba394238785b933915f66.

Reply:
Thanks @ziw-liu...I just tested this and it's working well.